docs/getting-started/playground-start.md

```diff
@@ -0,0 +1,105 @@
+---
+id: playground-start
+---
+
+# Gno Playground
+
+## Overview
+
+The Gno Playground is an innovative web-based editor and sandbox that enables developers to 
+interactively work with the Gno language. It makes coding, testing,
+and deploying simple with its diverse set of tools and features. Users can
+share code, run tests, and deploy projects to Gno.land networks, 
+making it the perfect tool to get started with Gno development.
+
+## Prerequisites
+
+- **Internet connection**
+- **A Gno.land compatible wallet**
+  - Currently, [Adena](https://www.adena.app/) is the go-to wallet for
```

Comment:
I would rephrase this as `...is the preferred wallet...`

Reply:
[0051394](https://github.com/gnolang/gno/pull/1773/commits/0051394bd6090098946bbfccaa0b67db0a9f7048)